components/Predict/PredictionForm.jsx

```diff
@@ -32,45 +59,56 @@ const PredictionForm = () => {
     try {
       await postPredictionRequest(payload);
 
-      const predictionRequests = await getPredictionRequests();
-      dispatch(setPredictionRequests(predictionRequests));
+      const { approvedRequestsCount } = await getPredictionRequests();
+      dispatch(setApprovedRequestsCount(approvedRequestsCount));
 
-      notifySuccess('Prediction requested');
+      notifySuccess('Question asked');
       form.resetFields();
     } catch (error) {
       notifyError('Request failed');
       console.error(error);
     } finally {
       setIsLoading(false);
+      setIsModalVisible(false);
     }
   };
 
+  const showModal = () => {
+    setIsModalVisible(true);
+  };
+
   return (
-    <Form form={form} layout="vertical" onFinish={handleSubmit}>
-      <Form.Item
-        label="Question"
-        name="question"
-        extra="Questions must expect a yes/no answer"
-        rules={[{ required: true, message: 'Please input a question' }]}
-      >
-        <TextArea />
-      </Form.Item>
-
-      <Form.Item
-        label="Resolution time"
-        name="resolution_time"
-        extra="This is when you receive your answer"
-        rules={[{ required: true, message: 'Please pick the resolution time' }]}
-      >
-        <DatePicker showTime />
-      </Form.Item>
-
-      <Form.Item>
-        <Button type="primary" htmlType="submit" loading={isLoading}>
-          Request
-        </Button>
-      </Form.Item>
-    </Form>
+    <>
+      <Button type="primary" onClick={showModal}>
+        Ask a question
+      </Button>
+      <Modal title="Ask a question" visible={isModalVisible} onCancel={() => setIsModalVisible(false)} footer={null}>
+        <Form form={form} layout="vertical" onFinish={handleSubmit}>
+          <Form.Item
+            label="Question"
+            name="question"
+            extra="Questions must expect a yes/no answer"
+            rules={[{ required: true, message: 'Please input a question' }]}
+          >
+            <TextArea />
+          </Form.Item>
+
+          <Form.Item
+            label="Closing date"
```

Comment:
It must be clear that "resolution_time" is the date we know for sure if the event happened or not. This might be different from the closing date of a market.
Right now, the "closing date" of the Omen market is the next day at 00:00 midnight. For example:
https://aiomen.eth.limo/#/0xf143562525b8464df76a5fbfb3edd3f1f7596a87
refers to an event happening on 20 th oct. So it will be closed at 21 oct 00:00 UTC